prowler/providers/azure/services/network/network_bastion_host_exists/network_bastion_host_exists.py

```diff
@@ -0,0 +1,28 @@
+from prowler.lib.check.models import Check, Check_Report_Azure
+from prowler.providers.azure.services.network.network_client import network_client
+
+
+class network_bastion_host_exists(Check):
+    def execute(self) -> Check_Report_Azure:
+        findings = []
+        for subscription, bastion_hosts in network_client.bastion_hosts.items():
+            if bastion_hosts == []:
+                status = "FAIL"
+                status_extended = (
+                    f"Bastion Host from subscription {subscription} does not exist"
+                )
+            else:
+                status = "PASS"
+                status_extended = (
+                    f"Bastion Host from subscription {subscription} exists"
```

Comment:
Please, add here the bastion hosts that are available

Reply:
Done!